doc/changes.rst

```diff
@@ -3,6 +3,13 @@ Changes
 
 .. module:: sksparse.cholmod
 
+v0.4
+------
+  * Support for 64-bit indices (type long) is added.
+  * Ordering method for cholesky decomposition is now choseable.
+  * The exception handling is refined.
```

Comment:
"64-bit indices (type long) are now supported."
"The ordering method for Cholesky decomposition is now choosable."
"Specific exceptions subclasses are now thrown for each error condition."